modules/ezcontent_node/modules/ezcontent_article/config/install/core.entity_form_display.node.article.default.yml

```diff
@@ -267,6 +277,7 @@ content:
       match_operator: CONTAINS
       size: 60
       placeholder: ''
+      match_limit: 10
```

Comment:
What is this change about ?

Reply:
This is a change introduced in core: https://www.drupal.org/node/3024684